tests/pdctl/store/store_test.go

```diff
@@ -327,4 +327,21 @@ func (s *storeTestSuite) TestStore(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(scene.Idle, Equals, 100)
 
+	// store check status
+	args = []string{"-u", pdAddr, "store", "check", "Offline"}
+	output, err = pdctl.ExecuteCommand(cmd, args...)
+	c.Assert(err, IsNil)
+	c.Assert(strings.Contains(string(output), "PANIC"), IsFalse)
```

Comment:
It's better to check whether the output contains the offline stores instead of `PANIC`.